From 8390c478600b852392cb116741b3cb239c94d123 Mon Sep 17 00:00:00 2001
From: Brad Fitzpatrick <bradfitz@golang.org>
Date: Wed, 15 Jan 2020 18:08:10 +0000
Subject: [PATCH] net/http/cgi: rename a test file to be less cute

My fault (from CL 4245070), sorry.

Change-Id: Ib95d3170dc326e74aa74c22421c4e44a8b00f577
Reviewed-on: https://go-review.googlesource.com/c/go/+/214920
Run-TryBot: Brad Fitzpatrick <bradfitz@golang.org>
TryBot-Result: Gobot Gobot <gobot@golang.org>
Reviewed-by: Emmanuel Odeke <emm.odeke@gmail.com>

Upstream-Status: Backport
[lz: Add this patch for merging the patch for CVE-2020-24553]
Signed-off-by: Li Zhou <li.zhou@windriver.com>
---
 src/net/http/cgi/{matryoshka_test.go => integration_test.go} | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename src/net/http/cgi/{matryoshka_test.go => integration_test.go} (100%)

diff --git a/src/net/http/cgi/matryoshka_test.go b/src/net/http/cgi/integration_test.go
similarity index 100%
rename from src/net/http/cgi/matryoshka_test.go
rename to src/net/http/cgi/integration_test.go
-- 
2.17.1

